src/service/overwritePersonaAttribute.ts

```diff
@@ -0,0 +1,10 @@
+import OverwritePersonaAttributeOptions from // tslint:disable-line:import-spacing
+  '../serviceFactory/options/OverwritePersonaAttributeOptions';
+import OverwritePersonaAttributeResult from // tslint:disable-line:import-spacing
+  '../serviceFactory/results/OverwritePersonaAttributeResult';
+import Config from './Config';
+
+export default (config: Config) =>
+  async (opts: OverwritePersonaAttributeOptions): Promise<OverwritePersonaAttributeResult> => {
+    return config.repo.overwritePersonaAttribute(opts);
```

Comment:
Use this in the facade, then we can use the repo options and result interfaces from the repo and delete the service options and result interfaces.

Reply:
So I take something from the result and put it in the expect? As opposed to writing a couple extra explicit asserts?